src/dmd/backend/fp.d

```diff
@@ -46,8 +46,6 @@ version (SPP) {} else
             enum HOST_IS_BSD = true;
         else version(DragonFlyBSD)
             enum HOST_IS_BSD = true;
-        else version(OpenBSD)
-            enum HOST_IS_BSD = true;
```

Comment:
`HOST_IS_BSD` no longer makes sense as a name here. Please change it to `HOST_USE_FMOD` and exclude OpenBSD from it.

Reply:
I'll do you one better. I just looked up if FreeBSD and DragonFly BSD have fmodl. FreeBSD has had it since 2008; DragonFly since 2013. I think it's time the whole special casing goes away.
Force pushed that change.